ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/handlers/JsonRpcObjectExecutor.java

```diff
@@ -76,7 +79,11 @@ private void handleJsonObjectResponse(
       try (final JsonResponseStreamer streamer =
           new JsonResponseStreamer(response, ctx.request().remoteAddress())) {
         // underlying output stream lifecycle is managed by the json object writer
-        lazyTraceLogger(() -> getJsonObjectMapper().writeValueAsString(jsonRpcResponse));
+        if (jsonRpcResponse.getType() != RpcResponseType.ERROR) {
+          lazyTraceLogger(() -> getJsonObjectMapper().writeValueAsString(jsonRpcResponse));
+        } else {
+          LOG.warn("RPC error: {}", getJsonObjectMapper().writeValueAsString(jsonRpcResponse));
+        }
```

Comment:
This could be pretty heavy on RPC nodes, I’m thinking of “Block not found” errors for example.

Reply:
Agree, this could potentially blow up the logs for a lot of users, even if they are genuine issues.
Maybe we need some sort of log throttling mechanism to go with it?

Would favour the more specific METHOD_NOT_FOUND but unless we also have throttling, I think we'd still need some confidence about the impact of this on a variety of usecases/networks.